File: arthrData/Todo/TodoEditItemModel.cs
using arthrData.Core;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace arthrData.Todo
{
    public class TodoItemEditModel
    {
        
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime? Deadline { get; set; }
        public bool IsDone { get; set; }
        public int Priority { get; set; }
        public int MastersiteId { get; set; }
        public List<MasterSite> MasterSites { get; set; }
        public int StatusId { get; set; }
        public List<Status> Status { get; set; }

    }
}

File: arthr.Web/Scripts/Models/TodoEditItemModel.ts
import { MasterSite } from './MasterSite';
import { Status } from './Status';

export class TodoItemEditModel {
    deadline: Date;
    description: string;
    isDone: boolean;
    mastersiteId: number;
    masterSites: MasterSite[];
    priority: number;
    status: Status[];
    statusId: number;
    title: string;
}
/*
    Debug Info:
    Unknown Types: MasterSite, Status
*/
